styled.d.ts

```diff
@@ -0,0 +1,44 @@
+import "styled-components";
+
```

Comment:
Are these not themes changes? Similar thing to what happened with me with an accidental merge maybe

Reply:
Commented below - it is a mistake it shouldn't be branched off from VFB-16 but now too many things are dependent on it - will fix before merging into main